bundles/c4c-mock-0.1.0/docs/meta.yaml

```diff
@@ -0,0 +1,9 @@
+docs:
+  - template:
+      displayName: "c4c-mock-0.1.0"
+      description: "SAP C4C Mock documentation"

```

Comment:
Please see the root meta.yaml  for the correct name, something with Cloud for Customers